src/schema.rs

```diff
@@ -0,0 +1,93 @@
+use std::path::{Path, PathBuf};
+
+use colored::{ColoredString, Colorize};
+use log::{error, info};
+use schemars::{generate::SchemaSettings, JsonSchema, SchemaGenerator};
+
+use crate::{config::Config, module::{repository::RepositoryConfig, ModuleConfig}, output::table::{table, Column}};
+
+pub const DEFAULT_PARENT: &str = "~/.local/share";
+pub const DEFAULT_DIR: &str = "/schemas";
```

Comment:
is this intended that it is written to `XDG_DATA_HOME/schemas`, with no mention of pusta?